eng/Versions.props

```diff
@@ -8,7 +8,7 @@
     <MajorVersion>3</MajorVersion>
     <MinorVersion>8</MinorVersion>
     <PatchVersion>0</PatchVersion>
-    <PreReleaseVersionLabel>1</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>2</PreReleaseVersionLabel>
```

Comment:
This looks like a bad merge here. You're targetting the 16.8 p1 branch now but this change is producing 16.8p2 binaries. Think you need to rebase this change on top of 16.8preview1 or somehow filter out this commit.